functions/exercise11/scriptSpec.js

```diff
@@ -0,0 +1,9 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getSecondLowestAndSecondGreatestNumbers*/
+
+describe("Getting second lovest and second gratest numbers with array", function () {
+    it("[1, 2, 3, 4, 5] is 2 and 4 ", function () {
+        var value = getSecondLowestAndSecondGreatestNumbers([1, 2, 3, 4, 5]);
```

Comment:
You should return object here as you have to provide two pieces of information - second lowest and largest.
So value name here is not the best.